### ISSUE TITLE
Provide FDS Version

### ISSUE THREAD

**Issue by TristanHehnen** [FireDynamics/fdsreader#52]
Hi,

could we keep track of the FDS version that was used to run the simulation results that are to be loaded? 
It could be stored in the simulation object directly (`fdsreader.Simulation()`) as `fds_version`.

Best,
Tristan

**Comment by JanVogelsang** [comment]
Hi, this is already implemented exactly as you suggested. See the [API reference](https://firedynamics.github.io/fdsreader/simulation.html).

**Comment by TristanHehnen** [comment]
Ah, my bad, should have read properly. Thanks!

**Comment by lu-kas** [comment]
Wait. I just wanted to use it, yet the variable `fds_version` is not available in the `Simulation` object.

In the source, the FDS version is determined with
https://github.com/FireDynamics/fdsreader/blob/e26beb7a627249c8da92bbec93b3abc69619bdef/fdsreader/simulation.py#L209

Yet, in (at least a recent) SMV file, the keyword is `FDSVERSION`:

```TITLE
 f001

FDSVERSION
FDS6.7.9-0-gec52dee-HEAD

ENDF
 f001.end

INPF
 f001.fds

REVISION
Revisio                                                                                                                                                                                                                                                        

CHID
 f001

SOLID_HT3D
  0

CSVF
 hrr
 f001_hrr.csv
[...]```

I will prepare a fix.

**Comment by JanVogelsang** [comment]
Could you also check if the key has been 'VERSION' in a prior FDS version (e.g. 6.7.5). To provide backwards compatibility, we could look for either 'VERSION' or 'FDSVERSION' to get the version.

**Comment by lu-kas** [comment]
Good point!

**Comment by lu-kas** [comment]
Fixed in #53
